tasks/main.yaml

```diff
@@ -0,0 +1,77 @@
+---
+- name: Install required pakages
+  package:
+    name: "{{ item }}"
+    state: present
+  with_items: "{{ capra_lvm_filesystem_packages }}"
```

Comment:
`package` accepts a list directly.

Reply:
Where is this documented?

Reply:
I figured out by reading code and lots of our other Ansible roles make use of that feature.